dask/tests/test_threaded.py

```diff
@@ -109,6 +110,19 @@ def test_dont_spawn_too_many_threads():
     assert after <= before + 8
 
 
+def test_dont_spawn_too_many_threads_CPU_COUNT():
+    before = threading.active_count()
+
+    dsk = {("x", i): (lambda: i,) for i in range(10)}
+    dsk["x"] = (sum, list(dsk))
+    for i in range(20):
+        get(dsk, "x")
+
+    after = threading.active_count()
+
+    assert after <= before + CPU_COUNT * 2
```

Comment:
For my own understanding, why is there a factor of 2 here?

Reply:
I don't know. It's a test I lifted from Distributed. @jcrist , do you know?